MobiFlight/Config/MuxDriver.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Linq;
+using System.Xml.Serialization;
+
+namespace MobiFlight.Config
+{
+    // MuxDriver is not really used as a full-fledged BaseDevice (though it could really well be);
+    // it is mainly embedded in other devices which are mux clients.
+    // However, it inherits from BaseDevice because:
+    // 1 - it uses many of the features of the base class in the very same capacity
+    // 2 - this way it can easily be returned to the role of full Device, should it be required
+    public class MuxDriver: BaseDevice
```

Comment:
rename MultiplexerDriver

Why do we need this separation at all in the UI side of MobiFlight? It seems to be adding additional complexity.

Reply:
Again, should have a look back, but I would say that all driver UI subpanels should necessarily refer to the same object with its own identity. What would be a sensible alternative? Otherwise incorporating the same information by repeating it over all InputMultiplexer objects makes little sense to me (beside being even more complex and calling for sync issues).